python/cugraph/cugraph/experimental/datasets/dataset.py

```diff
@@ -19,23 +19,30 @@
 
 
 class DefaultDownloadDir:
+    """
+    An object that is used to store the location of the dataset files. It is
+    shared across all dataset objects. Upon creation, it checks for the
+    existence of the environment variable RAPIDS_DATASET_ROOT_DIR. Otherwise,
+    defaults to <HOME>/.cugraph/datasets.
+    """
```

Comment:
It might be better to just briefly describe the class and how it's intended to be used, then let other parts of the code/comments describe the specifics with the env var, etc.
```suggestion
    """
    Maintains the path to the download directory used by Dataset instances.
    Instances of this class are typically shared by several Dataset instances
    in order to allow for the download directory to be defined and updated by
    a single object.
    """
```